supply_energy.md

```diff
@@ -0,0 +1,291 @@
+
+##### Traditional biomass
+
+Traditional biomass in GCAM is defined as the IEA's "primary solid biomass" product consumed by the residential sector, in selected regions where it is considered to be an important part of the energy system. The largest consumers of traditional biomass in 2010 were China, India, and Western Africa. The specific energy goods involved include firewood, agricultural residues, animal dung, and others; no effort is made to disaggregate the category into these consistuent parts, or to link the production volumes with the agriculture and land use module.
+
+
+### Energy Transformation
+
+Broadly, the energy transformation sectors in GCAM consist of all supplysectors between the [primary energy resources](supply_energy.html#resources) and the [final energy demands](demand_energy.html). The main energy transformation sectors highlighted in this documentation are [electricity](supply_energy.html#electricity), [refining](supply_energy.html#refining), [gas processing](supply_energy.html#en_gas_processing), [hydrogen production](supply_energy.html#hydrogen), and [district services](supply_energy.html#district-services). For more details, see [energy transformation](details_energy.html#energy-transformation)
+
+#### Electricity
+
+The GCAM electricity sector models the conversion of primary fuels (e.g., coal, gas, oil, bioenergy) to electricity. For most fuels, GCAM includes several different technology options (e.g., pulverized coal, coal IGCC, etc.). Individual technologies compete for market share based on their technological characteristics (conversion efficiency in the production of products from inputs), and cost of inputs and price of outputs. The cost of a technology in any period depends on (1) its exogenously specified non-energy cost, (2) its endogenously calculated fuel cost, and (3) any cost of emissions, as determined by the climate policy. The first term, non-energy cost, represents capital, fixed and variable O&M costs incurred over the lifetime of the equipment (except for fuel or electricity costs). For electricity technologies, GCAM reads in each of these terms and computes the levelized cost of energy within the model. For example, the non-energy cost of coal-fired power plant is calculated as the sum of overnight capital cost (amortized using a capital recovery factor and converted to dollars per unit of energy output by applying a capacity factor), fixed and variable operations and maintenance costs. The second term, fuel or electricity cost, depends on the specified efficiency of the technology, which determines the amount of fuel or electricity required to produce each unit of output, as well as the cost of the fuel or electricity. For more information, see [electricity](details_energy.html#electricity).
```

Comment:
- "(e.g., pulverized coal, coal IGCC, etc.)":  I generally prefer no un-explained acronyms, although integrated gasification combined cycle is a mouthful.
- I'm going to stop flagging these at the end of each section, but the links aren't working (at least on the GH render), so I can't tell if they're erroneous self-references or links to relevant pages elsewhere (most don't seem to correspond to sections within this supply_energy.md page).

Reply:
For your second point, it looks like this is an issue with the existing documentation, so I don't have anything to copy to fix it. I'm going to make a list of issues like this that we need to address later